src/Microsoft.Health.Fhir.Core/Features/Definition/SearchParameterDefinitionManager.cs

```diff
@@ -36,16 +36,21 @@ public SearchParameterDefinitionManager(IModelInfoProvider modelInfoProvider)
 
         public void Start()
         {
-            var builder = new SearchParameterDefinitionBuilder(
-                _modelInfoProvider,
-                "search-parameters.json");
+            if (!_started)
+            {
+                var builder = new SearchParameterDefinitionBuilder(
+                    _modelInfoProvider,
+                    "search-parameters.json");
+
+                builder.Build();
 
-            builder.Build();
+                _typeLookup = builder.ResourceTypeDictionary;
+                UrlLookup = builder.UriDictionary;
 
-            _typeLookup = builder.ResourceTypeDictionary;
-            UrlLookup = builder.UriDictionary;
+                List<string> list = UrlLookup.Values.Where(p => p.Type == ValueSets.SearchParamType.Composite).Select(p => string.Join("|", p.Component.Select(c => UrlLookup[c.DefinitionUrl].Type))).Distinct().ToList();
```

Comment:
@jackliums do you remember why we have this list that isn't used? I keep looking at it and wondering